VIDEO_3_SCRIPT.md

```diff
@@ -0,0 +1,43 @@
+# Video 3: What is AsyncAPI
+ 
+Hello everyone! Welcome!
+		
+This video will help you understand what AsyncAPI is and how to continue learning about this Open-Source project. 
+
+In this video, we'll talk about:
+ - What is AsyncAPI
+ - How did AsyncAPI come about
+ - What AsyncAPI brings 
+ 
+ Ready? Let's get started!
+
+## Definition of AsyncAPI
```

Comment:
when I look at the content of this section, I feel it is about AsyncAPI Initiative, not AsyncAPI Specification